{
  test.withException(new TestWithExceptionCallback(){
    @Override public void doTest() throws EntitlementRepairException {
    }
  }
,ErrorCode.ENT_REPAIR_NEW_EVENT_BEFORE_LAST_AO_REMAINING);
}
